Makefile

```diff
@@ -351,6 +351,9 @@ endif
 	PRIVATE_KEY=${ACCOUNT_PRIVKEY} make self-register-node peer_ids=$(shell eval ./scripts/get-hopr-address.sh "$(api_token)" "$(endpoint)")
 
 ensure-environment-is-set:
+ifeq ($(network),)
```

Comment:
This check is also done further down, which should then be removed.

Reply:
Some make commands were failing because they didn't provide the network argument. I noticed that the usage of the network argument was being used before checking its value. The first usage is in line 361
`network != jq '.environments."$(environment)".network_id // empty' packages/core/protocol-config.json`
And the next line it is where it is check.
At least that is what I understand from reading the code, not an expert in Makefile